src/components/users/confirm-password/confirmPasswordModel.ts

```diff
@@ -1 +1,8 @@
-export class ConfirmPasswordModel { }
+import { LocalStyles } from "@paperbits/common/styles";
+
+export class ConfirmPasswordModel {
+    /**
```

Comment:
Formatting of the jsdocs is a bit shifted. It should be like this:
```
    /**
     * Widget local styles.
     */
```
Just in case: https://jsdoc.app/about-getting-started.html